src/lib.rs

```diff
@@ -101,7 +102,14 @@ mod util;
 ///        // Defaults to Some(13), option-stripping is performed:
 ///        #[builder(default = Some(13))]
 ///        z: Option<i32>,
+///
+///        // Accepts params `(x: f32, y: f32)`
+///        #[builder(setter(transform = |x: f32, y: f32| Point { x, y }))]
+///        w: Point,
 ///    }
+///
+///    #[derive(Default)]
```

Comment:
I get this this is just an example, but... why?

Reply:
Because all fields are `default` by default, and I intend to change as little code as possible.